modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -44,7 +44,7 @@ endif::[]
 
 NOTE: The underlying environment variable that {prod-short} server uses is `CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT`.
 
-WARNING: By default, only one workspace in the same {orch-namespace} can be running at one time. See xref:configuring-the-number-of-workspaces-a-user-can-run.adoc[].
+WARNING: By default, only one workspace in the same {orch-namespace} can be running at one time. See xref:configuring-the-number-of-workspaces-that-a-user-can-run.adoc[].
```

Comment:
Just looking at this sentence without any context:
"Only one workspace can be running" because the reader can allow it to run or because the reader is the one who makes it run?

Reply:
I believe the user can only run one workspace because that's the default configuration. In this file, I only updated the reference. I did not attempt to rewrite the content.

Reply:
"By default, only one workspace in the same {orch-namespace} can be running at one time."
could also be:
"By default, users are only allowed to run a single workspace per {orch-namespace}, but this limit can be increased by an administrator to allow multiple concurrent workspaces to run in parallel, or to run unlimited workspaces."

See https://github.com/eclipse-che/che-server/blob/main/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties where `che.limits.user.workspaces.run.count=1` is mentioned.